event-adapter/pkg/apiserver/installer/context/context.go

```diff
@@ -20,9 +20,14 @@ import (
 	"k8s.io/apiserver/pkg/endpoints/request"
 )
 
-// resourceInformation holds the resource and subresource for a request in the context.
+// resourceInformation holds the resource and method for a request in the context.
```

Comment:
What is method?

I think the name 'subresource' is a more abstract concept, and api machinery operates on this level of abstraction, so it would make more sense to use this word. I'm not sure if I understand all of this correctly, so please explain what is the reasoning behind calling this field 'method' and whether or not it is consistent with another field named 'resource'.

Reply:
From offline discussion - `method` is a method of http request, e.g. GET or POST. It is however a little bit confusing, since it shouldn't be a part of resource specification. Can you find some more appropriate naming? For example, change `resourceInformation` struct to `requestInformation`

Reply:
Done